src/Storefront/Page/Product/Review/ProductReviewsLoadedCompatibilitySubscriber.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Page\Product\Review;
+
+use Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewsLoadedEvent as CoreProductReviewsLoadedEvent;
+use Shopware\Core\Framework\Feature;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Storefront\Framework\Page\StorefrontSearchResult;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @internal
+ *
+ * @deprecated tag:v6.6.0 will be re<p></p>moved without replacement, since \Shopware\Storefront\Page\Product\Review\ProductReviewsLoadedEvent will be removed
```

Comment:
```suggestion
 * @deprecated tag:v6.6.0 will be removed without replacement, since \Shopware\Storefront\Page\Product\Review\ProductReviewsLoadedEvent will be removed
```